docs/cc.cozycloud.announcements.md

```diff
@@ -0,0 +1,11 @@
+[Table of contents](README.md#table-of-contents)
+
+# Announcements remote doctype
+
+This `cc.cozycloud.announcements` remote doctypes are used to get announcement to display in the Home application. The API is a project based on [Strapi](https://strapi.io/).
+
+Home needs access to two routes:
+- public-announcements : to get a list of announcements for a language and certain channels
+- uploads : to get images contained in each announcement
```

Comment:
I would remove the Home mention, it is true now, but who knows in the future? Other apps might use it, and/or it could disappear from the Home. So I would replace "home" by something like "consuming app"

Reply:
Yes for sure, its removed